package.json

```diff
@@ -33,8 +33,9 @@
     "@commitlint/cli": "^8.3.5",
     "@commitlint/config-conventional": "^8.3.4",
     "@commitlint/prompt": "^8.3.5",
+    "@ethersproject/contracts": "^5.0.5",
```

Comment:
not sure why need this tho, was having this error:

![Screenshot 2020-10-09 at 1 29 26 PM](https://user-images.githubusercontent.com/4774314/95546599-97200500-0a33-11eb-8793-f8d3b6c23e92.png)